ocean_provider/run.py

```diff
@@ -25,11 +25,27 @@ def get_version():
 
 @app.route("/")
 def version():
+    """
```

Comment:
This doc comment should describe what the entire function is doing, not just the changes you made. Please use a compliant description if you choose to add one.

I would suggest something like:
Displays data necessary for using the provider, including software, version, network url, provider address and a list of services endpoints that may be used.